base_station/gui/src/components/Cameras.vue

```diff
@@ -1,115 +1,30 @@
 <template>
   <div class="wrap">
-    <div class="box header">
-      <h3>Cameras</h3>
-      <div class="spacer"></div>
-      <div class="comms">
-        <ul id="vitals">
-          <li v-for="connection, i in connections" :key="i"><CommIndicator v-bind:connected="connection" v-bind:name="'Pi'+(i+1)"/></li>
-        </ul>
-        </div>
+    <h3>Cameras</h3>
+    <div>
+      <Checkbox v-bind:name="'Microscope'" v-on:toggle="toggleMicroscopeCam()" ref="micro"/>
```

Comment:
idk i plan on removing the microscope button... i would add to the todo to see about microscope stuff

Reply:
maybe don't, idk i think we should take advantage of being able to always identify it ideally soon, wouldn't make much sense imo to delete then add in

Reply:
@catkims  is working on camera stuff now, so she could do it. But we should hold off on that for now?

Reply:
Should we make this a draft to keep from running travis unnecessarily?

Reply:
smart, yes will convert to draft